### PR TITLE
Normative: Update RegExp property aliases per Unicode v13

### PR DESCRIPTION
**Pull request by mathiasbynens**
Tests: https://github.com/tc39/test262/pull/2526

Ref. #1897 & #1896.

### CODE REVIEW

File: table-binary-unicode-properties.html [hunk 1]
```diff
@@ -158,23 +158,38 @@
       <td><a href="https://unicode.org/reports/tr51/#Emoji_Properties">`Emoji`</a></td>
     </tr>
     <tr>
-      <td>`Emoji_Component`</td>
+      <ul>
```
Comment (bakkot) [inline table-binary-unicode-properties.html:161]: The markup is messed up here - these need to be inside of a `<td>`. (See the [preview](https://deploy-preview-1939--ecma262-snapshots.netlify.app/#table-binary-unicode-properties).)

**Comment by michaelficarra** [comment]
Rebased and fixed.

**Comment by mathiasbynens** [comment]
What happened here? The change that landed is completely different from [what my original PR was](https://github.com/tc39/ecma262/commit/c1c1299244cd6e57fd47623636c7fe890c57c615). I'll post the original diff here in case that link stops to work:

```
From c1c1299244cd6e57fd47623636c7fe890c57c615 Mon Sep 17 00:00:00 2001
From: Mathias Bynens <mathias@qiwi.be>
Date: Wed, 11 Mar 2020 14:35:51 -0300
Subject: [PATCH] Normative: Update Unicode property lists per Unicode v13

https://unicode.org/versions/Unicode13.0.0/
---
 table-unicode-script-values.html | 36 ++++++++++++++++++++++++++++++++
 1 file changed, 36 insertions(+)

diff --git a/table-unicode-script-values.html b/table-unicode-script-values.html
index 652c4288b..b5451aeca 100644
--- a/table-unicode-script-values.html
+++ b/table-unicode-script-values.html
@@ -205,6 +205,15 @@
       </td>
       <td>`Cham`</td>
     </tr>
+    <tr>
+      <td>
+        <ul>
+          <li>`Chorasmian`</li>
+          <li>`Chrs`</li>
+        </ul>
+      </td>
+      <td>`Chorasmian`</td>
+    </tr>
     <tr>
       <td>
         <ul>
@@ -278,6 +287,15 @@
       </td>
       <td>`Devanagari`</td>
     </tr>
+    <tr>
+      <td>
+        <ul>
+          <li>`Dives_Akuru`</li>
+          <li>`Diak`</li>
+        </ul>
+      </td>
+      <td>`Dives_Akuru`</td>
+    </tr>
     <tr>
       <td>
         <ul>
@@ -558,6 +576,15 @@
       </td>
       <td>`Kharoshthi`</td>
     </tr>
+    <tr>
+      <td>
+        <ul>
+          <li>`Khitan_Small_Script`</li>
+          <li>`Kits`</li>
+        </ul>
+      </td>
+      <td>`Khitan_Small_Script`</td>
+    </tr>
     <tr>
       <td>
         <ul>
@@ -1359,6 +1386,15 @@
       </td>
       <td>`Warang_Citi`</td>
     </tr>
+    <tr>
+      <td>
+        <ul>
+          <li>`Yezidi`</li>
+          <li>`Yezi`</li>
+        </ul>
+      </td>
+      <td>`Yezidi`</td>
+    </tr>
     <tr>
       <td>
         <ul>
```

**Comment by mathiasbynens** [comment]
My bad, was looking at the wrong PR for the wrong description! Ignore my last comment.

**Pull request merged**